OpenTracing/OpenTracing.OpenTracing/ISpan.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace OpenTracing
+{
+    public interface ISpan<T> 
```

Comment:
fine to leave them out for now, but before merging we will want some clear comments for everything in ISpan and ITracer.